_data/teams.yml

```diff
@@ -20,7 +20,7 @@
       lastname:    Dean
       location:    Pittsburgh
       division:    Warehouse & Logistics
-      description: Jon is a web software engineer and pipes everything to /dev/null, the only true web scale. He's composed of 60% water, 20% coffee, 19% lager and 1% other. Hails from Pittsburgh.
+      description: Jon cares about great software but about providing a tangible value more. He's composed of 60% water, 20% coffee, 12% IPA and 8% scotch or bourbon. Hails from Pittsburgh.
```

Comment:
Oops, thanks

On Friday, January 30, 2015, Joe Cannatti <notifications@github.com> wrote:

> In _data/teams.yml
> <https://github.com/stitchfix/stitchfix.github.io/pull/65#discussion_r23854041>
> :
>
> > @@ -20,7 +20,7 @@
> >        lastname:    Dean
> >        location:    Pittsburgh
> >        division:    Warehouse & Logistics
> > -      description: Jon is a web software engineer and pipes everything to /dev/null, the only true web scale. He's composed of 60% water, 20% coffee, 19% lager and 1% other. Hails from Pittsburgh.
> > +      description: Jon cares about great software but about providing a tangible value more. He's composed of 60% water, 20% coffee, 12% IPA and 8% scotch or bourbon. Hails from Pittsburgh.
>
> s/cares about/cares not only about/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stitchfix/stitchfix.github.io/pull/65/files#r23854041>
> .
>


-- 
*Jon Dean*
Principal Engineer
*Stitch Fix | **your partner in personal style*

*technology.stitchfix.com <http://technology.stitchfix.com>*

jon@stitchfix.com
412-596-8348 (Eastern Time)